src/Microsoft.Health.Fhir.Api/Controllers/FhirController.cs

```diff
@@ -221,10 +222,11 @@ public async Task<IActionResult> Read(string type, string id)
         public async Task<IActionResult> SystemHistory(
             [FromQuery(Name = KnownQueryParameterNames.At)] PartialDateTime at,
```

Comment:
Not your code, but I just realized that the `at` parasmeter is an instant type. Filed new [bug](https://github.com/microsoft/fhir-server/issues/463) for this.